core/src/main/java/io/aiven/klaw/config/ManageDatabase.java

```diff
@@ -123,9 +124,6 @@ public class ManageDatabase implements ApplicationContextAware, InitializingBean
   @Value("${klaw.superadmin.default.password}")
   private String superAdminDefaultPwd;
 
-  @Value("${klaw.version}")
```

Comment:
can we also remove this from application.properties ?

Reply:
Yeah its a good question I left it "in case" it was used elsewhere but honestly yeah I should just remove it will take a look at that